components/meta-packages/history/history

```diff
@@ -1258,6 +1258,8 @@ service/network/load-balancer/pen@0.18.0,5.11-2013.0.0.1
 service/network/samba/samba36@3.6.25,5.11-2015.0.2.3
 service/postrun@1.0,5.11-2015.0.2.0
 service/storage/avs/cache-management@0.5.11,5.11-2022.0.1.0
+shell/whichh2.21-2020.0.1.2
+shell/whichh2.21-2023.0.0.2
 slim_install@0.1,5.11-2015.0.2.1
```

Comment:
The actual version is 2.21-2020.0.1.1, so the 2nd line is superfluous.

Reply:
Besides of that there is missing `@`.